src/test/java/teammates/test/cases/ui/browsertests/InstructorCommentsPageUiTest.java

```diff
@@ -146,6 +147,26 @@ private void testScripts() throws Exception {
         commentsPage.showCommentsForStatus("private");
         commentsPage.verifyHtmlMainContent("/instructorCommentsPageShowCommentsForPrivate.html");
     }
+    
+    /**
+     * Tests to ensure the panels can be collapse / expand.
+     */
```

Comment:
I don't think this comment say anything that's not already apparent from the signature.